mk/spksrc.service.start-stop-status

```diff
@@ -29,103 +29,138 @@ call_func ()
 
 start_daemon ()
 {
+    declare -i i=0
     if [ -z "${SVC_QUIET}" ]; then
-        if [ -z "${SVC_KEEP_LOG}" ]; then
-            date > ${LOG_FILE}
-        else
-            date >> ${LOG_FILE}
-        fi
+        # Keep logs by default
```

Comment:
Please do not change such a default behaviour!
- packages might write a lot of log output
- log is never deleted and old log is kept on package updates
- logs are neigther rotated nor compressed by our generic service support
- to report issues, the log since service start is sufficient and I assume that users will include the full log to issues reported here

If you have services that really should keep the logs, it would be better, we could write logs to `/var/log/packages` were those might get rotated (not verified yet).
IMHO the option `SVC_QUIET = no` is more for package development than for production. Currently no package defines `SVC_QUIET`, this shows that deleting the logs on service start is a good default behaviour.

Reply:
I'll revert to the original behavior.  Although while playing with the package I find it really anoying to remove thel previous log, always.  The best would be to incorporate a logrotate by default.  Page 107 for Syslog Config mentions something about that similar to (but that's for another time and PR):
```
"syslog-config": {
"logrotate-relpath": "etc/<appname>-logrotate.conf"
}
```

Beyond that, any thought otherwise on the proposed changes?